cypress/integration/examples/utilities.spec.js

```diff
@@ -0,0 +1,136 @@
+/// <reference types="cypress" />
+
+context('Utilities', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/utilities')
```

Comment:
The beforeEach hook should visit the Book Store app URL, not 'https://example.cypress.io/utilities'. Update the URL accordingly.